float32 leftEyeX
float32 leftEyeY
float32 rightEyeX
float32 rightEyeY
